src/tests/components/post/MakePost.test.jsx

```diff
@@ -7,22 +7,4 @@ describe("Post box Component", () => {
     const component = shallow(<MakePost />);
     expect(component).toMatchSnapshot();
   });
-
-  it("should have a text area", () => {
-    const component = shallow(<MakePost />);
-    expect(component.find("textarea")).toHaveLength(1);
-  });
-
-  it("should show the modal after clicking on image icon", () => {
```

Comment:
This test wasn't actually doing anything - when changing the expected value to false it still passed 😃